content/07.pathogenesis.md

```diff
@@ -46,8 +46,12 @@ One severe example includes reports of cytokine storm-like responses in patients
 #### Reproduction Number and Dynamics of Transmission
 
 Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
-The basic reproduction number, R~0~, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
+In models the quantities of basic reproduction (or reproductive) number and the timescale of infection have a large influence on the population level dynamics of epidemics.
```

Comment:
```suggestion
The quantities of basic reproduction (or reproductive) number and the timescale of infection have a large influence on the population-level dynamics in epidemic models.
```
It reads a little weird to me to have the prepositional phrase at the beginning and then a different one at the end (I feel like I don't know what we're talking about, exactly). Is this what you meant? If this doesn't work, perhaps there's another way to streamline it?

Reply:
```suggestion
The quantities of the basic reproduction (or reproductive) number and the effective reproduction number have a large influence on disease population dynamics.
```

Reply:
yes I think this sentence needs to be clarified - "in models" not needed and I don't know what is meant by "timescale of infections"

Reply:
I made a paraphrase from a modern infectious disease modeling textbook, I'll go back & re-check the exact wording. I think they mean for example the average time between start of primary infection and start of secondary infection -- in other words the generation time. (also I think maybe mathematically since R0 is unitless it can only theoretically describe shapes but you also need something indicating a timescale / with units of time; relatedly, you can make a simple early epidemic growing graph by raising R0 to the exponent of #generations = time divided by generation time & also the same textbook has a related equation using the SIR with simple demographics for early epidemics as I(t) ~ I(0) exp([R0-1][gamma+mu]t) where here generation time is like inverse (gamma+mu). People usually just mention R0 because it's important & probably for other reasons I don't even know of, but I was trying to give a mathematically complete picture so that it makes more sense to people outside the field.

Reply:
@rando2 @alavendelm @rdvelazquez  I am still learning how to use this platform & this is a bit of complicated pull request for me to do while I'm learning. I may have messed up in trying to fix this. Here's basically, what I'd like to do and would like assistance with. I went back to the exact wording of the textbook again & would like to suggest one possible way to fix this sentence which would be closer to the wording of the textbook & also include a part of the textbook that is related to the original text emphasis on R0 (& what @alavendelm was bringing back with his suggestion); however, I'm not sure about using such a strong word as "govern" which is what the textbook uses & that's why I wanted to put something weaker (like "influence" but that may be too weak; maybe we could use "describe the" rather than "govern"??).:
-The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@ISBN:9780691116174].
OR
-The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) describe the population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@ISBN:9780691116174].
Would one of you maybe insert one of these more accurate sentences as a suggestion and then I can "commit suggestion" or some other process you suggest? Thank you for allowing for a bit of messiness while I'm learning!

Reply:
@yemarshall git and GitHub are not easy to learn at first. You've done great jumping in and figuring things out! 

I don't know what process you have been using for making edits so this suggestion might not be applicable but, just in case you weren't aware of it, you can use GitHub.com to make edits to this pull-request at any time. Once you opened the pull-request it is basically saying that you want to incorporate changes from your `patch-9` branch into greenelab's `master` branch... so any changes that you make to your `patch-9` branch while this pull-request is open will show up here (and likewise, any changes that you make here via the `commit suggestion` will show up in your `patch-9` branch). 

To make a change to your `patch-9` branch you can just edit this file: https://github.com/yemarshall/covid19-review/edit/patch-9/content/07.pathogenesis.md and click the green `commit changes` button at the bottom. To get to this file I clicked the blue `yemarshall:patch-9` at the top of this page and then navigated to `content` then `07.pathogenesis.md`, then clicked the pencil icon that says "edit this file" when you hover over it.

There are many ways to use git/github, what I described above isn't the best or preferred way of doing things it's just another thing to be aware of. If it helps you great, if not feel free to disregard. Thanks again for all your additions and discussions; it's been really informative for me to follow along. Happy editing!